ui/app/templates/vault/cluster/access/mfa/enforcements/enforcement/index.hbs

```diff
@@ -60,47 +60,54 @@
 </Toolbar>
 
 {{#if (eq this.tab "targets")}}
-  {{#each @model.targets as |target|}}
-    <LinkedBlock
-      class="list-item-row"
-      @disabled={{not target.link}}
-      @params={{union (array target.link) target.linkModels}}
-      data-test-target={{target.title}}
-    >
-      <div class="level is-mobile">
-        <div class="level-left">
-          <div>
-            <Icon @name={{target.icon}} />
-            <span class="has-text-weight-semibold has-text-black">
-              {{target.title}}
-            </span>
-            <div class="has-text-grey is-size-8">
-              <code>
-                {{target.subTitle}}
-              </code>
+  {{#if (gt @model.targets.length 0)}}
```

Comment:
You could simplify this to `@model.targets`. In the template it will evaluate to `false` for an empty array.